README.md

```diff
@@ -4,57 +4,89 @@ Crusher generates statistics from raw events.
 
 ## Requirements
 
-* REview 0.39.0 or higher
-* Giganto 0.21.0 or higher
+- REview 0.39.0 or higher
+- Giganto 0.21.0 or higher
 
 ## Usage
 
 You can run Crusher by invoking the following command:
 
 ```sh
-crusher path/to/config.toml
+crusher \
+ --cert <CERT_PATH> --key <KEY_PATH> --ca-certs <CA_CERTS_PATH> \
+ <SERVER_NAME>@<SERVER_IP>:<SERVER_PORT>
 ```
 
-where `config.toml` is a configuration file in TOML format.
+To run Crusher with a local configuration file, with multiple CA certificates:
+
+```sh
+crusher \
+ -c <CONFIG_PATH> \
+ --cert <CERT_PATH>
+ --key <KEY_PATH> \
+ --ca-certs <CA_CERTS_PATH> \
+ --ca-certs <CA_CERTS_PATH> \
+ <SERVER_NAME>@<SERVER_IP>:<SERVER_PORT>
+```
+
+In these commands:
+
+- `<CONFIG_PATH>` is the path to your TOML configuration file.
+- `<CERT_PATH>` is the path to the certificate file for the current module.
+- `<KEY_PATH>` is the path to the private key file for the current module.
+- `<CA_CERTS_PATH>` is a CA certificate files. Multiple CA certificates can be
+  specified by repeating the `--ca-certs` option.
+- `<SERVER_NAME>` is the name of the manager server, and
+  `<SERVER_IP>:<SERVER_PORT>` is the IP address and port number of the manager
+  server.
+  Ensure that `<SERVER_NAME>` matches the DNS name specified in the certificate.
```

Comment:
I think we don't need a line break for L42. Could you concat it after L41 with a space please?

Reply:
The lines were concatenated.